templates/lxc-ubuntu.in

```diff
@@ -343,6 +344,12 @@ download_ubuntu()
     fi
     packages_template="${packages_template},$(echo $langpacks | sed 's/ /,/g')"
 
+    if [ -n "$variant" ]; then
+        debootstrap_parameters="$debootstrap_parameters --variant=$variant"
+    fi
+    if [ $variant = 'minbase' ]; then
```

Comment:
$variant must be "$variant" if you don't want to get a shell error when $variant isn't set.